roofit/roofitcore/inc/RooArgSet.h

```diff
@@ -75,35 +75,75 @@ class RooArgSet : public RooAbsCollection {
 	    const RooAbsArg& var5, const RooAbsArg& var6, 
 	    const RooAbsArg& var7, const RooAbsArg& var8, 
 	    const RooAbsArg& var9, const char *name="");
+  /// Create from any kind of container that can be iterated and holds pointers
+  /// to a type that derives from RooAbsArg.
+  template <class Input_t,
+      class Container_t = typename std::remove_reference<Input_t>::type,
+      class = typename std::enable_if<std::is_convertible<typename Container_t::value_type, const RooAbsArg*>::value>::type >
+  RooArgSet(Input_t inputs, const char* name = "") :
+  RooAbsCollection(name)
+  {
+    for (const auto elm : inputs) {
+      add(elm);
```

Comment:
Can't you just deref the pointers here, instead of having to add pointer overloads in the base class?